news/4.31/platform.html

```diff
@@ -142,6 +142,32 @@ <h2>Preferences </h2>
     <td id="StylingThemes" class="section" colspan="2">
     <h2>Themes and Styling </h2>
     </td>
+  </tr>
+    <tr id="full_text_in_view_tabs"> <!-- https://github.com/eclipse-platform/eclipse.platform.ui/issues/735 -->
+    <td class="title">Full Text in View Tabs</td>
+    <td class="content">
+        <p>
+            A new preference has been introduced to enhance the user experience for managing views within perspectives. With this preference, users have more control over how view tabs are displayed.
```

Comment:
why "in perspectives"? Can't we leave this addition out and write:
```suggestion
            A new preference has been introduced to enhance the user experience for managing views. With this preference, users have more control over how view tabs are displayed.
```